qutebrowser/browser/qutescheme.py

```diff
@@ -435,6 +435,20 @@ def qute_settings(url):
     return 'text/html', html
 
 
+@add_handler('bindings')
+def qute_bindings(url):
+    """Handler for qute://bindings View qute bindings."""
```

Comment:
This should have a period after `qute://bindings`. Also, "View qute bindings." isn't really descriptive, something like "View keybindings." or so would be better.